### DISCUSSION TITLE
The memory  limits of ceph-mds recommend?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12897] [Q&A]
**
![image](https://github.com/rook/rook/assets/13997135/8a97c2b8-c77e-480d-ad1b-5a6a2e86cf98)
**
The memory usage of the ceph-mds overhead 8G.

**Comment by travisn** [comment]
Have you set the memory limits on the mds in the [filesystem CR](https://github.com/rook/rook/blob/master/deploy/examples/filesystem.yaml#L103-L110)? If so, I believe the mds will try to honor those limits. The limits will really depend on usage. 4Gi is a common limit, although 8Gi may also be needed for heavy load.

**Comment by zhucan** [comment]
yes. the memory usage out of 8GiB
